### PR TITLE
check bkmgit's change on GitHub Actions

### LINKED ISSUE #1: Enable GitHub actions to build a pdf from LaTeX sources

### PR DESCRIPTION
**Pull request by zigen**
the original PR is #2 which resolves #1 
to run the GitHub Actions workflow, someone in `sfc-aqua` must commit and push it to the repo.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - name: Add comment to PR with workflow output
+      env:
+          PR: ${{ github.event.pull_request.number }}
+          EVENT_NUMBER: ${{ github.event.number }}
+          URL: ${{ github.api_url }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OWNER: ${{ github.repository_owner }}
+          REPO: ${{ github.repository }}
+          RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          MESSAGE: Download [pdf](${{ github.server_url }}/${{ github.repository_owner }}/LaTeX-PDFs/raw/main/${{ env.SHORT_SHA }}/Quantum-Communications-${{ env.SHORT_SHA }}.pdf) or [artifacts](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})
```
Comment (bkmgit) [inline .github/workflows/build.yaml]:
```suggestion
          MESSAGE: Sign in to GitHub to download  [zipped pdf artifact](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})
```
This is a more accurate description.

**Comment by bkmgit** [comment]
Thanks for the fixes.  GitHub stores all artifacts as zipped folders.  May wish to set a maximum time to retain artifacts as storage is limited. It is possible to put the pdf in the repository or another repository, see https://github.com/orgs/LaTeX-Action-Test/repositories but as the pdf files are quite heavy may want to re-write git history to remove commits with older pdf files.

**Comment by zigen** [comment]
that's right. current artifact retention time is set to 90 days (maximum for public repository).
the pdf file size is 9.4MB, and the issue comment embeddable maximum file size is 10MB for free org
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files
and it will become bigger...

anyway, now we can download the zipped PDF in PR. I'll merge this and improve it in another PR. 
Thank you @bkmgit !

**Comment by zigen** [comment]
wait, we need a review and approval to merge it.

**Comment by bkmgit** [comment]
Will close other pull request https://github.com/sfc-aqua/Overview-of-Quantum-Communications-E/pull/2
Storage starts at 500 Mb. One can also upload the files to another storage service if that is desired.

**Review by rdviii — APPROVED** [review]
Looks fine, assuming it works -- I haven't tested but assume bkm & zigen have.

**Pull request merged**